packages/react/avatar/src/Avatar.tsx

```diff
@@ -120,7 +120,7 @@ AvatarFallback.displayName = FALLBACK_NAME;
 function useImageLoadingStatus(src?: string) {
   const [loadingStatus, setLoadingStatus] = React.useState<ImageLoadingStatus>('idle');
 
-  React.useEffect(() => {
+  React.useLayoutEffect(() => {
```

Comment:
```suggestion
  useLayoutEffect(() => {
```

We should use the isomorphic safe package for this:

`import { useLayoutEffect } from '@radix-ui/react-use-layout-effect';`

Reply:
Done